app/views/issues/_reports.html.erb

```diff
@@ -0,0 +1,19 @@
+<% reports.each do |report| %>
+  <div class="reports">
+    <div style="float:left">
+      <%= link_to user_thumbnail(report.user), :controller => :user, :action =>:view, :display_name => report.user.display_name %>
+    </div>
+    <%= t(".reported_by_html", :user_name => report.user.display_name, :user_url => url_for(:controller => :user, :action => :view, :display_name => report.user.display_name)) %> <br/>
+    <span class="deemphasize">
+      <%= t(".updated_at", :datetime => l(report.updated_at.to_datetime, :format => :friendly)) %>
+    </span>
+    <br/>
+    <span class="deemphasize">
+      <%= t ".category", category: report.category %>
```

Comment:
I'm tempted to get rid of this as a separate line and merge into the first line as "Reported as %{category} by %{user}" to save a bit of space and rid of the line with the ugly "Category:" prefix.